dependencies.yaml

```diff
@@ -43,6 +43,7 @@ test_requires:
   perl(Test::Warnings):
   perl(Test::MockModule):
   perl(Test::Most):
+  html-xml-utils:
```

Comment:
Why this doesnt work?

Reply:
Let's leave it for a follow up ticket as we don't use this properly yet